client/src/m_menu.cpp

```diff
@@ -1638,10 +1706,17 @@ static void M_PlayerTeamChanged (int choice)
 
 static void SendNewColor(int red, int green, int blue)
 {
-	char command[24];
+	char colorcommand[24];
+	char customcolorcommand[30];// Acts 19 quiz
+	int colorpreset = D_ColorPreset(cl_colorpreset.cstring());// Acts 19 quiz
 
-	sprintf(command, "cl_color \"%02x %02x %02x\"", red, green, blue);
-	AddCommandString(command);
+	sprintf(colorcommand, "cl_color \"%02x %02x %02x\"", red, green, blue);
```

Comment:
So, as a rule, if you need to change a line that has a `sprintf` in it, please change it to something that's not `sprintf`.  The function is incredibly dangerous and there are safer alternatives, both standard and specific to our codebase.

In this instance, there is no reason for the buffer to be fixed-length, so we have a function called `StrFormat` that operates on `std::string` that I would recommend switching to.